_posts/2024-04-11-sc-planning.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "2024 Steering Committee Strategic Planning Meeting Recap"
+tags: [steering committee]
+author: Abbey Roelofs
+---
+
+In February, the [Steering Committee](https://us-rse.org/about/steering-committee/) continued the tradition of yearly strategic planning meetings. Like [last year](https://us-rse.org/2023-03-29-sc-planning/), we focused on the direction and areas of focus for US-RSE over the next 3-5 years.
+
+We started by reflecting on the highlights and successes of the past year, including the receipt of the Sloan Foundation grant, the creation of a new [Code of Conduct](https://us-rse.org/about/code-of-conduct/), hiring our first two paid staff members, hosting many events including our [first US-RSE Conference](https://us-rse.org/usrse23/), producing resources such as a [Hiring Guidebook](https://zenodo.org/records/8329337) and [Career Guide](https://zenodo.org/records/10073233), and starting the new [Community Funds](https://us-rse.org/funds-and-awards/) and [Community Awards](https://us-rse.org/community-awards/) initiatives. 
```

Comment:
can you change the zenodo links to be the DOIs?

Reply:
https://doi.org/10.5281/zenodo.8329337 and https://doi.org/10.5281/zenodo.10073233 respectively...

Reply:
Sure, they're updated now.